### PR TITLE
Update approved-price-identifiers.md

### PR DESCRIPTION
**Pull request by henrystats**

### CODE REVIEW

File: docs/uma-tokenholders/approved-collateral-currencies.md [hunk 1]
```diff
@@ -36,6 +36,18 @@ sidebar_label: Approved Collateral Currencies
 |BAL| 8 |[UMIP-72](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-72.md)
 |xSUSHI| 22 | [UMIP-67](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-67.md)
 |XIO| 1500 | [UMIP-71](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-70.md)
```
Comment (smb2796) [inline docs/uma-tokenholders/approved-collateral-currencies.md:38]: Looks good @henrystats! Do you mind updating the collateral currency final fees that were changed in [UPP-3](https://github.com/UMAprotocol/UMIPs/blob/master/UPPs/upp-3.md)?

Comment (henrystats) [reply docs/uma-tokenholders/approved-collateral-currencies.md:38]: yes boss! on it!

Comment (henrystats) [reply docs/uma-tokenholders/approved-collateral-currencies.md:38]: I've updated the final fees, can you please approve so I can merge?

**Review by smb2796 — APPROVED** [review]

File: docs/uma-tokenholders/approved-collateral-currencies.md (hunk 1 above)
Comment (smb2796) [reply docs/uma-tokenholders/approved-collateral-currencies.md:38]: Thanks!

**Pull request merged**